composer.json

```diff
@@ -6,18 +6,18 @@
     "require": {
         "php": "~8.1",
         "ext-amqp": "*",
-        "symfony/amqp-messenger": "^6.0",
-        "symfony/http-kernel": "^5.4|^6.0",
-        "symfony/messenger": "^6.0"
+        "symfony/amqp-messenger": "^6.0|^7.0",
+        "symfony/http-kernel": "^5.4|^6.0|^7.0",
+        "symfony/messenger": "^6.0|^7.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^3.4",
-        "phpro/grumphp": "^1.5.0",
-        "phpstan/phpstan": "^1.4",
-        "phpstan/phpstan-webmozart-assert": "^1.0",
-        "phpunit/phpunit": "^9.5.28",
+        "friendsofphp/php-cs-fixer": "^3.63",
+        "phpro/grumphp": "^2.6",
```

Comment:
did you run grum on whole project after this changes?

Reply:
I did now.